### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by aimalamiri**
## Design the tasks list layout
1. Design the tasks
2. Added the icons
3. Design the text input
4. Design the task checkbox using a checkbox generator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (FrederickMih) [inline README.md:41]: - [ ] This `MIT` License link is broken. Please kindly rectify this issue.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @aimalamiri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- Linters checks are all passing :heavy_check_mark: 
- PR has a title and description :heavy_check_mark: 

## Required Changes ♻️

- [ ] I got this error when I ran the command `npm install`

Screenshot from 2022-04-27 04-57-06

Please check your package.json file and fix this issue.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (aimalamiri) [reply README.md:41]: I added the MIT.md license file.

**Review by dasileker — APPROVED** [review]
### **Hi @aimalamiri,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**